DEVELOPMENT.md

```diff
@@ -53,6 +53,8 @@ This is done by running `yarn changeset`, which will guide you through all of th
 
 Commit the resulting changeset file with your other changes and push it up. This can happen at any time in the lifecycle of the branch.
 
+Changesets isn't smart enough to detect dependent changes and version accordingly. If you are only changing code in the `@chromatic-com/shared-e2e` package, you will need to explicitly include the `chromatic-com/playwright` and `chromatic-com/cypress` packages as packages to version-bump as well (assuming the change affects both packages).
+
```

Comment:
@tevanoff come to think of it, do we even need to include the `@chromatic-com/shared-e2e` package as a package to version-bump, or can we always leave it out? I think we have been inconsistent in bumping it when changes are made in its code.

Reply:
@skitterm yeah it's a good question. The only reason we'd need to bump it currently is if we want there to be an entry in the shared changelog. We haven't been consistent about that so I could go either way.

Reply:
Sounds good! I think we'd always want to publish `@chromatic-com/e2e-shared` if publishable code inside it changes, that way we know from its changelog what specifically changed in the shared package when. I've pushed up a doc change.